examples/vite-core/src/wallet.ts

```diff
@@ -2,10 +2,8 @@ import { FedimintWallet } from '@fedimint/core-web'
 
 const wallet = new FedimintWallet()
 
-// try {
-wallet.open()
-// } catch (e) {
-//   console.warn('Failed to open wallet', e)
-// }
+function init(onOpen: (boolean)) {
+   kwallet.open().then(onOpen)
+}
```

Comment:
reverting this change due to some errors... will readd in follow-up

```suggestion
wallet.open()
```